src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
Remove the 'salt' field from the UserResponseDto, as it's not a good practice to expose user salt in response DTOs.